src/lib.rs

```diff
@@ -252,6 +252,8 @@ struct CommonMarkOptions {
     use_explicit_uri_scheme: bool,
     default_implicit_uri_scheme: String,
     alerts: AlertBundle,
+    /// Whether to present a mutable ui for things like checkboxes
```

Comment:
Does this belong in CommonMarkOptions, or should it be somewhere else?

Reply:
It's fine for it to live there